src/snapshots/new_nu_parser__test__lexer.snap

```diff
@@ -1,21 +0,0 @@
----
```

Comment:
Why was this removed?

Reply:
It seems like whatever was generating this snapshot was removed at some point (haven't looked at the history). We have https://github.com/nushell/new-nu-parser/blob/main/src/snapshots/new_nu_parser__test__lexer%40int.nu.snap now instead